src/Employer/Employer.Web/wwwroot/javascripts/application.js

```diff
@@ -104,4 +104,38 @@ $(function() {
     sfa.navigation.init();
 });
 
-   
\ No newline at end of file
+/* -----------------------
+Character count behaviour
+-------------------------- */
+characterCount = function(n) {
+  var i = $(n)
```

Comment:
👨‍🔬 bit of a science equation here. Could we have slightly more meaningful names to the variables. Or has this been lifted from a  library.